src/config/Config.ts

```diff
@@ -155,6 +155,12 @@ export interface ClientConfig {
      */
     properties?: Properties;
 
+    /**
+     * Metrics config. With this config, you enable collecting the client metrics and sending them to the cluster.
```

Comment:
How about: "Using this configuration, you can enable client metrics collection"?